src/main/kotlin/com/github/devcordde/devcordbot/listeners/NameWatcher.kt

```diff
@@ -36,9 +36,11 @@ fun Kord.addNameWatcher(bot: DevCordBot) {
     }
 }
 
+private val NAME_VALIDATION_REGEX = "[a-z0-9äüö]{3}.*".toRegex(RegexOption.IGNORE_CASE)
+
 private suspend fun Member.sanitizeNameIfNeeded(bot: DevCordBot) {
     val sanitizedName = effictiveName.sanitize()
-    if (sanitizedName != effictiveName) {
+    if (sanitizedName.relevantChars() != effictiveName.relevantChars() && sanitizedName.matches(NAME_VALIDATION_REGEX)) {
```

Comment:
Change the order of these checks. Otherwise, Names with less than 3 characters cause an exception.